README.md

```diff
@@ -9,11 +9,13 @@ C API to Qt for Qlift
 - `ENABLE_SHARED`: Enable or disable building qlift-c-api as a shared library. Default is ON.
 - `ENABLE_STATIC`: Enable or disable building qlift-c-api as a static library. Default is OFF.
 
-To install as a shared library in ```/usr/local```:
+To install as a shared library in ```/usr/local``` (default install directory):
 ```sh
 git clone https://github.com/Longhanks/qlift-c-api
 cd qlift-c-api
-cmake CMAKE_INSTALL_PREFIX=/usr/local .
+mkdir build
+cd build
+cmake ..
 make
 sudo make install # root access may not be nessasary if you have /usr/local set up right
```

Comment:
Blast it, I can never spell necessary right...

By set up right I mean ```rwxrwxr-x``` and ```root:staff```. The goal there is to allow staff users to make changes to /usr/local without needing to be root. That way you don't have to sudo install scripts/makefiles that are huge and you don't want to read over. Apparently there is a way to make it so that all things that staff users put in there have that owner/group too, but I don't remember. Would you like me to just remove that comment?